movement/watch_faces/complication/tarot_face.c

```diff
@@ -32,6 +32,23 @@
 #include <string.h>
 #include "tarot_face.h"
 
+#define DEFAULT_MAJOR_ARCANA_ONLY true
+#define SHOW_REVERSED_CARDS true
+#define TAROT_MIN_CARDS_TO_DRAW 1
```

Comment:
I like the default of 3 since it covers a "normal" spread but also can be used for a 1 or 2-card spread without changing this value (since you can just ignore the 2nd/3rd cards depending on your desired spread)